storage/people.json

```diff
@@ -0,0 +1 @@
+["{\"id\":36,\"name\":\"thierry\",\"age\":23}","{\"id\":422,\"name\":\"Eugene\",\"age\":44}","{\"id\":500,\"name\":\"a\",\"age\":3}","{\"id\":825,\"name\":\"a\",\"age\":3}","{\"id\":547,\"name\":\"omwom\",\"age\":44}","{\"id\":26,\"name\":\"a\",\"age\":3}","{\"id\":598,\"name\":\"Ofafa\",\"age\":26}","{\"id\":403,\"name\":\"Ofafa\",\"age\":26}","{\"id\":613,\"name\":\"a\",\"age\":3}","{\"id\":248,\"name\":\"a\",\"age\":1}","{\"id\":882,\"name\":\"a\",\"age\":0}","{\"id\":607,\"name\":\"a\",\"age\":1}","{\"id\":268,\"name\":\"q\",\"age\":1}","{\"id\":944,\"name\":\"q\",\"age\":1}","{\"id\":64,\"name\":\"q\",\"age\":1}","{\"id\":862,\"name\":\"q\",\"age\":1}","{\"id\":230,\"name\":\"q\",\"age\":1}","{\"id\":580,\"name\":\"q\",\"age\":1}","{\"id\":809,\"name\":\"w\",\"age\":2}","{\"id\":19,\"name\":\"q\",\"age\":1}","{\"id\":970,\"name\":\"q\",\"age\":1}","{\"id\":249,\"name\":\"q\",\"age\":1}","{\"id\":792,\"name\":\"w\",\"age\":2}","{\"id\":434,\"name\":\"q\",\"age\":2}","{\"id\":484,\"name\":\"c\",\"age\":0}","{\"id\":604,\"name\":\"q\",\"age\":2}","{\"id\":393,\"name\":\"q\",\"age\":1}","{\"id\":385,\"name\":\"w\",\"age\":2}","{\"id\":872,\"name\":\"q\",\"age\":1}","{\"id\":700,\"name\":\"w\",\"age\":2}","{\"id\":533,\"name\":\"w\",\"age\":2}","{\"id\":366,\"name\":\"q\",\"age\":1}","{\"id\":966,\"name\":\"s\",\"age\":0}"]
```

Comment:
- [ ] As I mentioned in my above `inline-comment` the same is happening here, once you fix the `books` JSON file formatting and save the data appropriately then take a look into this file too, and implement the things that I mentioned earlier.